traveler_app/src/main/java/com/guestlogix/traveler/activities/HomeActivity.java

```diff
@@ -49,7 +50,7 @@ protected void onCreate(Bundle savedInstanceState) {
         this.profile = Profile.storedProfile(this);
 
         if (profile != null) {
-            Traveler.identify(profile.getTravelerId());
+            Traveler.identify(profile.getTravelerId(), null);
```

Comment:
Lets create another method for convenience that doesnt take in anything and assumes null.. Its just nicer :)

Reply:
changed and pushed